Source/BoundaryConditions/WarpXEvolvePML.cpp

```diff
@@ -56,25 +57,92 @@ WarpX::DampPML (int lev, PatchType patch_type)
             const Box& tby  = mfi.tilebox(By_nodal_flag);
             const Box& tbz  = mfi.tilebox(Bz_nodal_flag);
 
-            WRPX_DAMP_PML(tex.loVect(), tex.hiVect(),
```

Comment:
Could you also remove the macro `WRPX_DAMP_PML` and `WRPX_DAMP_PML_F`, in the file `WarpX_f.H`, and remove the file `PML_routines.F90` altogether?